src/modules/Crowbar.rb

```diff
@@ -195,7 +202,15 @@ def Read
       @conduit_map = network["conduit_map"] || []
 
       @template_crowbar = json2hash(@crowbar_file)
-      @users = @template_crowbar["attributes"]["crowbar"]["users"] rescue {}
+
+      begin
+        crowbarrc = IniFile.load("/etc/crowbarrc")
```

Comment:
So. AFAIR this is not the  "YaST way" of accessing ini files, we should create an agent and access through SCR. At least it was it this way in the past.

However, I'm fine with this, as yast-crowbar is special and the only reasonable feature why this SCR module was introduced in the past (AFAIK) was a possibility to remotely access different machine that needs editing. I think this was never implemented and even if it was, yast-crowbar does not need it.

Reply:
@lslezak, would you have time to comment?

I'm fine approving it

Reply:
Using SCR was the old YCP way. With Ruby it's preferred to either use the native libraries and parsers or to use the augeas library.

But I think here you should use the `@crowbarrc_file` variable which contain also the `Installation.destdir` prefix.

```suggestion
        crowbarrc = IniFile.load(@crowbarrc_file)
```

Reply:
well, if inifile works for you, then it is fine. Known limitation is that it does not work over chroot ( during installation ) and also when you switch SCR during tests ( probably also not case here ).